packages/html/src/radio/radio-group.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import { classNames } from '../misc';
+
+export const RADIOGROUP_CLASSNAME = `k-radio-list`;
+
+const states = [];
+
+const options = {};
+
+export type KendoRadioGroupProps = {
+    layout?: 'vertical' | 'horizontal';
+};
+
+const defaultProps = {
+    layout: 'vertical',
+};
+
+export const RadioGroup = (
+    props: KendoRadioGroupProps &
+        React.HTMLAttributes<HTMLUListElement>
+) => {
+    const {
+        layout = defaultProps.layout,
+        ...other
+    } = props;
+
+    return (
+        <ul
+            {...other}
+            className={classNames(
+                props.className,
+                RADIOGROUP_CLASSNAME,
+                {
+                    [`k-list-${layout}`]: layout === 'horizontal',
```

Comment:
This will render only _**k-layout-horizontal**_ class (when layout is set to horizontal). I think we should render k-list-vertical class by default, as well as when layout is set to vertical